test/e2e/kustomize_provisioner_test.go

```diff
@@ -0,0 +1,315 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	"github.com/operator-framework/rukpak/internal/provisioner/kustomize"
+)
+
+var _ = Describe("kustomize provisioner bundledeployment", func() {
```

Comment:
Should we add a test for the default behavior when no spec.config has been specified?

Reply:
Yes, we should.  I'll add one.